sys/linux/bpf_prog.txt

```diff
@@ -1,6 +1,9 @@
 # Copyright 2023 syzkaller project authors. All rights reserved.
 # Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
 
+include <uapi/linux/bpf.h>
```

Comment:
cc @tarasmadan fyi

one potential task for better testing infra that can run end-to-end tests and have access to linux sources is to run `make extract` and verify that checked-in descriptions are valid